docs/maintainers/maintainers-guide.md

```diff
@@ -0,0 +1,152 @@
+- **Sales Leads**: Identify contributors whose engagement could potentially be turned into sales opportunities.
+- **Potential Hires**: Observe contributors who exhibit skill and commitment, making them prime candidates for hiring.
+- **Alumni Contributors**: Keep an eye on contributors who have decreased their engagement or left the project.
+
+### Creating a New List
+1. To create a new list, click the "Lists" tab on the Insights' hub. You will be redirected to a page where you can create your new list.
+2. Click on the 'New List' button to start a new list.
+3. Give your list a name.
+4. Choose your page visibility. You can choose to make your list public or private.
+4. Add contributors to your list by searching for their GitHub username, syncing your GitHub Team, or by importing your GitHub Following. 
+
+![team sync gif](../../static/gif/team-sync.gif)
+
+## Using Your List
+Once you have created your list, you will be redirected to your list page. Here you will see a dashboard with an overview  of the repositories and the contributors who have contributed to them. 
+
+There are three tabs that provide you with more information on these repositories.
+
+### Overview
+The Overview view gives you a high-level view of the contibutors in your list, including the total number of commits in the last 30 days and the types of contributors on your list: active, new, and alumni.
+
+### Activity
+The Activity tab gives you a graph view with more detailed information on each of the contributors, including type of activity, repositories they've contributed to, and how they compare to each other. 
+
+You can filter your list by all contributors, active contributors, new contributors, and alumni contributors.
+
+![list graph](../../static/img/list-graph.png)
+
+### Contributors
+The contributors view, gives you the list of the contributors and includes information like activity level, last repository contributed to, the date of their last contribution, time zone, programming languages used, and number of contributions.
+
+ To learn more about these features, see [Understanding Lists Data](#understanding-lists-data).
+
+## Highlights: Connecting and Recognizing Contributions
+Leveraging OpenSauced's Highlights feature is a great way for maintainers to recognize and showcase the valuable contributions made to your open source projects, to curate and share key achievements, contributions, and milestones, and bring visibility to the hard work of your community. 
+
+Highlights can be used to:
+- publicize issues on your repository that you'd like to match with contributors
+- share Dev.to blog posts you've written
+- celebrate PRs to your project
+
+### Creating a New Highlight
+There are two ways to create a new highlight:
+1. On the [main feed](https://app.opensauced.pizza/feed), click the input that says "Post a highlight to show your work!"
```

Comment:
For someone who visit OpenSauced for the first time, they might click the link to go to the feed. But probably the next time they visit, they won't find any "feed" word anywhere. Should we do this instead?

```suggestion
1. On the top bar, click the "Highlights" tab, then click the input that says "Post a highlight to show your work!"
```